example/webpack.config.js

```diff
@@ -67,6 +67,9 @@ module.exports = {
     publicPath: "/",
     filename: "rn-skia-example.bundle.js",
   },
+  devServer: {
+    historyApiFallback: true,
+  },
```

Comment:
This allows `webpack-dev-server` to know that when we navigate to a path e.g. `localhost:8080/filters` it should serve `index.html` ([doc](https://webpack.js.org/configuration/dev-server/#devserverhistoryapifallback)). Otherwise we would get `Cannot get /filters`